src/runtime/cmd/netmon/netmon.go

```diff
@@ -170,7 +170,7 @@ func newNetmon(params netmonParams) (*netmon, error) {
 
 func (n *netmon) cleanup() {
 	os.RemoveAll(n.storagePath)
-	n.netHandler.Delete()
```

Comment:
Is this a change in the interface? If so, could you announce it in the commit message to explain the change?

Reply:
Good point. I added a note on the commit message now.